docs/user/troubleshooting.md

```diff
@@ -79,13 +79,13 @@ The most important thing to look at on the bootstrap node is `bootkube.service`.
 
 ### etcd Is Not Running
 
-etcd is started and managed by the Kubelet as a static pod. This requires a newer Kubelet which started shipping with version 47.29 of Red Hat CoreOS. The OS version can be checked using the following command:
+etcd is started and managed by the Kubelet as a static pod. This requires a newer Kubelet which started shipping with version 47.29 of Red Hat Enterprise Linux CoreOS. The OS version can be checked using the following command:
```

Comment:
Should we drop this paragraph through the next?  I don't expect anyone is running RHCOS <27.29 anymore.  We probably want to keep "During the bootstrap process, the Kubelet may emit errors like..." and later.

Reply:
Sure. Updating.

Reply:
Removed.